telemetry/definitions/commonDefinitions.json

```diff
@@ -5405,6 +5405,16 @@
                 }
             ]
         },
+        {
+            "name": "ide_editCodeFile",
```

Comment:
will this ever be passive? like opening a existing project or something?

Reply:
good point. added passive:true